src/components/shared/card/card.scss

```
@@ -0,0 +1,228 @@
+}
+
+.card__text.microSize {
+  font: var(--font-style-bold) 33px/50px Rubik;
+}
+
+.card__text.nanoSize {
+  font: var(--font-style-bold) 22px/35px Rubik;
+}
+
+.card__startText,
+.card__endText {
+  display: inline;
+}
+
+.card__input {
+  height: 60px;
+  max-width: 160px;
+  padding-left: 20px;
+  vertical-align: middle;
+  font: var(--font-size-21)/var(--line-spacing-32) var(--font-family-rubik);
+  letter-spacing: var(--character-spacing-0);
+  background: var(--blue-light-gray) 0 0 no-repeat padding-box;
+  border: 1px solid var(--blue-light-gray);
+  border-radius: 15px;
+  outline:none;
+}
+
+.card__input.smallSize {
+  height: 50px;
+}
+
+.card__input.microSize,
+.card__input.nanoSize  {
+  height: 30px;
+}
+
+.card__translateText {
+  margin-bottom: 10px;
+  width: 600px;
+  height: 40px;
+  border-bottom: 1px solid var(--bg-green);
+  font: var(--font-size-21)/var(--line-spacing-32) var(--font-family-rubik);
+  letter-spacing: var(--character-spacing-0);
+  text-align: left;
+  color: var(--gray);
+}
+
+.card__wordType {
+  width: 600px;
+  height: 20px;
+  margin: auto auto 55px;
+  font: var(--font-size-11)/var(--line-spacing-17) var(--font-family-rubik);
+  letter-spacing: var(--character-spacing-0);
+  color: var(--gray);
+}
+
+.card__handlers {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.card_btnDelete {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 70px;
+  height: 30px;
+  font: Medium var(--font-size-15)/var(--line-spacing-32) var(--font-family-rubik);
+  letter-spacing: var(--character-spacing-0);
+  color: var(--red);
+  text-align: left;
+  cursor: pointer;
+
+  &:hover {
+  border-bottom: 2px solid var(--red);
+  }
+}
+
+.card_btnTransfer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 180px;
+  height: 30px;
+  font: Medium var(--font-size-15)/var(--line-spacing-32) var(--font-family-rubik);
+  letter-spacing: var(--character-spacing-0);
+  color: var(--dark-blue);
+  cursor: pointer;
+
+  &:hover {
+    border-bottom: 2px solid var(--dark-blue);
+  }
+}
+
+.card_btnShowAnswer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 170px;
+  height: 30px;
+  background: var(--blue-light-gray) 0 0 no-repeat padding-box;
+  box-shadow: 0 2px 1px var(--dark-gray);
+  border-radius: 15px;
+  font: Regular var(--font-size-15)/var(--line-spacing-32) var(--font-family-rubik);
+  letter-spacing: var(--character-spacing-0);
+  color: var(--blue);
+  text-align: center;
+  cursor: pointer;
+
+  &:hover {
+    background-color: #d0d2d7;
+  }
+}
+
+.card__bottomContent {
+  margin: 20px 60px 0 60px;
+}
+
+.card__wordTranslate {
+  font: var(--font-style-bold) 17px/32px Rubik;
+  letter-spacing: 0;
+  color: var(--dark);
+  text-align: left;
+}
+
+.card__bottomContentBtn {
+  display: flex;
+  justify-content: space-around;
+  margin-top: 15px;
+  height: 30px;
+  overflow: hidden;
+}
```

как вариант, элементы одного блока можно записать вложенными в него, чтобы была видна структура:
```suggestion
.card {
  &__bottomContent {
    ...
  }
  &__wordTranslate {
    ....
  }
  ...
}
```